### PR TITLE
Adds Floor Lights (and makes lights not have random ranges)

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Adds Floor Lights. (Sprites originally from https://github.com/tgstation/tgstation/pull/77594, modified to fit our construction system).
image
These are different from light tiles, they do not take up the entire tile but they can also be hit like normal lights. A frame for them takes 3 metal (tube frames take 2, bulb frames take 1). This makes them slightly more expensive, but they are more versatile and provide a lighting middle ground between tubes and bulbs, they have a lighting range of 6 where tubes and bulbs have 8 and 4 respectively.

Floor lights can be constructed with 3 metal, 1 wire, a screwdriver, and 1 light bulb.

This PR also removes the (rarely known) mechanic of player lightbulbs having a random brightness range, as this interfered with how floor bulbs worked.

This PR also resprites slime lights, which previously used the awful "floor light" sprite we had. This sprite has been ported from TGstation also.

(Also fucking refactors a lot of light code, and removes firebulbs (they were never used))

### Mapping Changes
Box
image
Delta
image

## Why It's Good For The Game

Allows for more creative expression and allows for lighting up large areas like Atmospherics without using those huge light tiles.

## Images of changes

image
image
image
image
image

https://github.com/user-attachments/assets/20e717c5-e0e4-4e15-bea9-4849a168c775

## Testing

Built a lot of lights. Destroyed a lot of lights. Made sure I could still construct and deconstruct other wall items like fire alarms. Can also construct non-floor light fixtures and place lights in them. Couldn't place floor-lights in walls.

Ran into a LOT of issues that required a LOT of fixing.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>
  
  
image

## Changelog

:cl:
add: Floor lights, like wall lights but in the floor. They can be constructed with 3 metal.
del: Lightbulbs no longer sometimes have a randomized light range.
tweak: Box and Delta have had their light-tiles replaced with floor lights.
imageadd: Glowing Slime Extract has a new sprite.
/:cl:

### CODE REVIEW

**Review by lewcc — APPROVED** [review]
LGTM

**Review by DGamerL — APPROVED** [review]
Can't see anything wrong with it

File: code/modules/lighting/lighting_atom.dm [hunk 1]
```diff
@@ -99,8 +99,11 @@
 	cut_overlay(exposure_overlay)
 
 	if(glow_icon && glow_icon_state)
-		glow_overlay = image(icon = glow_icon, icon_state = glow_icon_state, dir = dir, layer = 1)
-		glow_overlay.plane = LIGHTING_LAMPS_PLANE
+		glow_overlay = image(icon = glow_icon, icon_state = glow_icon_state, dir = dir, layer = -2)
```
Comment (lewcc) [inline code/modules/lighting/lighting_atom.dm:102]: Does changing the layer here impact how glare works for other light types?

**Review by lewcc — COMMENTED** [review]
Taking a closer look at this.
Does this have a broken state, or is it just on or off? Is there an easy way to get the bulb out or break the light, similar to other bulb-powered sources, or will you just need to break the fixture?

File: code/modules/lighting/lighting_atom.dm (hunk 1 above)
Comment (Contrabang) [reply code/modules/lighting/lighting_atom.dm:102]: Nope

**Review by Christasmurf — APPROVED** [review]
Icons looking good

**Pull request merged**